java/arcs/sdk/jvm/BaseParticle.kt

```diff
@@ -11,6 +11,9 @@
 
 package arcs.sdk
 
+typealias JvmEntity = arcs.jvm.storage.api.JvmEntity
+typealias JvmEntitySpec<T> = arcs.jvm.storage.api.JvmEntitySpec<T>
+
```

Comment:
KDoc would be good for these, also maybe moving them to an `arcs/sdk/jvm/Entity.kt` file?